src/getting-started.md

```diff
@@ -392,7 +392,7 @@ If the pool properly starts you should see the following log lines:
 ### 3. Start **Translator (tProxy) JN**
 
 Once the SV2 pool is running, let's run the tProxy that will facilitate communication between the pool and a SV1 miner.
-Differently from the Config C, in this case the **tProxy** will be acting as a **Job Negotiator (JN)**, so it will select transactions locally and send them to the **Pool-side JN**
+Differently from the Config C, in this case the **tProxy** will be acting as a **job declarator (JN)**, so it will select transactions locally and send them to the **Pool-side JN**
```

Comment:
```suggestion
Differently from the Config C, in this case the **tProxy** will be acting as a **Job Declarator (JD)**, so it will select transactions locally and send them to the **Pool-side JDS**
```

Reply:
https://github.com/stratum-mining/stratumprotocol.org/pull/182/commits/82a443ac23604eb511c76f8b8a20b2d14494bd04